README.md

```diff
@@ -22,6 +22,15 @@
 - Creation of worker nodes on AWS, Digitalocean, Openstack, Azure, Google Cloud Platform, VMWare Vsphere, Linode, Hetzner cloud and Kubevirt (experimental)
 - Using Ubuntu, CoreOS/RedHat ContainerLinux or CentOS 7 distributions ([not all distributions work on all providers](/docs/operating-system.md))
 
+## Supported Kubernetes versions
+Machine controller tries to follow as close as possible the Kubernetes version
+[support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/).
+
+Currently supported K8S versions are:
+- 1.17
+- 1.16
+- 1.15
```

Comment:
Why is 1.18 not supported? 1.18 came out a few weeks ago and we successfully use it for 1.18 clusters in KubeOne since then.

Also, should we have 1.15 on the list since it's not supported by the upstream anymore?

Reply:
Depends if we still support it or not I guess, I'm ok to add 1.18 if we support it.